### DISCUSSION TITLE
2020/06/solved-net-core-azure-ad-in-docker-container-incorrectly-uses-an-non-https-redirect-uri/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#673] [Specific Post Comments]
# 2020/06/solved-net-core-azure-ad-in-docker-container-incorrectly-uses-an-non-https-redirect-uri/

I’ve hit this problem several times and only recently figured it out.

https://seankilleen.com/2020/06/solved-net-core-azure-ad-in-docker-container-incorrectly-uses-an-non-https-redirect-uri/

**Comment by armbrat** [comment]
Thank you for figuring this out.

**Comment by DarylGraves** [comment]
In my situation, adding these lines didn't immediately fix the issue but I could see something had changed as the error was no longer visible in the browser, only in the dotnet application's terminal output.

In my situation **I also had to set "ForwardedHeaders_Enabled" to true** - You can do this by either adding this to your appsettings.json or by passing it through as an Environmental Variable (in Docker, that's `-e "ForwardedHeaders_Enabled=true"`)

Sean, a massive THANK YOU for this post - I've been pulling my hair out over this and without this I probably would've given up.

**Reply by SeanKilleen** [reply]
I'm glad it helped! Thanks for sharing your additional tip :)
